docs/pages/connect-your-client/tsh.mdx

```diff
@@ -849,6 +849,33 @@ DEBU [TSH]       Self re-exec command: tsh [status --format=json]. tsh/aliases.g
 ...
 ```
 
+## Debug logs
+
+Adding the `--debug` flag to a command or setting the `TELEPORT_DEBUG` env var to `1` makes tsh
+print debug logs to standard output.
+
+<Admonition type="note" >
```

Comment:
Since this `Admonition` is longer than the rest of the text in this section, I'd recommend either:
- Removing the tags and making the content regular body text (if we expect most readers to benefit from this text)
- Adding this to a `details` element instead of an `Admonition` (if most readers are going to skip this text)

Reply:
@ptgott I see. I used `Admonition` because I wanted this to be visible (`details` are easy to miss and don't show up when you search the page in your browser) and at the same time make it clear that the commands are relevant only for macOS. Perhaps I should introduce a new subsection instead?